ArduPlane/commands.cpp

```diff
@@ -124,6 +124,7 @@ void Plane::update_home()
         }
     }
     barometer.update_calibration();
+    ahrs.resetHeightDatum();
```

Comment:
This calls update_calibration on the barometer, so the previous baro needs to be removed, or this will probably defeat the point.

Reply:
Further investigation shows that each core calls this, so this should be fine. We will be calling this 1 + EKF2 cores + EKF3 cores now, but otherwise should be fine :)

Reply:
as discussed on mumble, it is safe to call twice, and ensures that DCM is updated